.github/workflows/update-phan-stubs.yml

```diff
@@ -58,12 +58,15 @@ jobs:
       - name: Create commit and PR
         if: steps.changes.outputs.needed == 'true' && steps.changes.outputs.has-branch == 'false'
         env:
-          GH_TOKEN: ${{ secrets.API_TOKEN_GITHUB }}
+          GH_TOKEN: ${{ secrets.STUB_UPDATE_TOKEN_GITHUB }}
```

Comment:
This is from https://github.com/Automattic/jetpack/pull/42256. We should merge that first.